TestResultSummaryService/plugins/DataManagerPerf.js

```diff
@@ -16,12 +16,14 @@ module.exports.onBuildDone = async (task, { testResultsDB, logger }) => {
                 if ( Array.isArray(childBuildList) && childBuildList.length > 0 ) {
                     jdkDate = childBuildList[0].jdkDate;
                     javaVersion = childBuildList[0].javaVersion;
+                    nodeRunDate = childBuildList[0].nodeRunDate;
+                    nodeVersion = childBuildList[0].nodeVersion;
                 }
                 //loop into the child build list
                 for ( let childBuild of childBuildList ) {
                     //loop into each child's tests info.
                     const {name, variant, benchmarkMetricsCollection} = DataManagerAggregate.aggDataCollect(childBuild)
-                    if (name === null || variant === null || jdkDate === null || javaVersion === null || benchmarkMetricsCollection === null ) {
+                    if (name === null || variant === null || (javaVersion == null && nodeVersion == null) || (jdkDate == null && nodeRunDate == null) || benchmarkMetricsCollection === null ) {
```

Comment:
- `==` and `===` have a different meaning. Please do not change the original logic. 
- nodeRunDate and nodeVersion will not be null, so the code can be changed as following:

```
if (name === null || variant === null || (javaVersion === null && !nodeVersion) || (jdkDate === null && !nodeRunDate) || benchmarkMetricsCollection === null ) {
```